src/treemodel.cpp

```diff
@@ -455,11 +468,17 @@ void TreeModel::SetupChild(TreeItem *child, const QJsonObject & event)
     child->SetData(COL::User, event["user"].toString());
     child->SetData(COL::Key, event["k"].toString());
     bool hasArtData = event.contains("a");
+    bool hasErrorCode = event.contains("e");
     if (hasArtData)
     {
         // 0xE2978F = BLACK CIRCLE
         child->SetData(COL::ART, QString::fromUtf8("\xE2\x97\x8F"));
     }
+    if (hasErrorCode)
+    {
+        // 0xE2978F = BLACK CIRCLE
```

Comment:
Perhaps a different symbol to make it easier to differentiate ART and error code.

Reply:
Changed to black square.